app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -6,24 +6,24 @@ import { Container, Header } from "neetoui/layouts";
 
 import EmptyState from "components/Common/EmptyState";
 
-import { NOTES as notes } from "./constants";
+import Block from "./Block";
+import { NOTES } from "./constants";
 import DeleteAlert from "./DeleteAlert";
-import Note from "./Note";
-import NotesMenu from "./NotesMenu";
+import Menu from "./Menu";
 import NewNotePane from "./Pane/Create";
 
 const Notes = () => {
   const [showNewNotePane, setShowNewNotePane] = useState(false);
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
   const [searchTerm, setSearchTerm] = useState("");
   const [showMenu, setShowMenu] = useState(false);
-  const [notesList, setNotes] = useState(notes);
-  const [selectedNoteId, setSelectedNoteId] = useState();
+  const [notesList, setNotesList] = useState(NOTES);
```

Comment:
You don't have to suffix with the keyword `List` in order to denote that the resource is a collection.

Using plural variable name suggests the same:
```suggestion
  const [notes, setNotes] = useState(NOTES);
```